### DISCUSSION TITLE
Github issue search too broad when NX report is included

### DISCUSSION THREAD

**Discussion by mwarger** [nrwl/nx#17267] [Feature Requests]
I want to ask if the `NX report` structure can be changed.  The issue I see is as follows:

If I'm heading to the Github repo and want to find open issues related to `expo`, I can put expo in the search box.  The results come back, and most of the issues that come back are not related to expo.  Why is this?  One of the common requests (from the bug template, afaik) is running the NX report.  This lists all first-party plugins.  Here's an example that returns during the search but has nothing to do with expo: https://github.com/nrwl/nx/issues/14790.  In this issue, @nrwl/expo is listed in the NX report as "Not Found".  I believe the fact that this keyword is in the issue is including it in the search results.  I think it would be helpful if this NX report only listed plugins that were found with their versions.  If it's not on the list, you can still deduce that it's "Not Found" without needing the text to say so.  This could cut down on search keyword results for unrelated plugins.

If this should be a bug or feature request, please let me know.  I'm curious about what others think about this.

**Comment by meeroslav** [comment]
You can filter issues by label and then search for `expo` - e.g. https://github.com/nrwl/nx/issues?q=is%3Aopen+is%3Aissue+label%3A%22scope%3A+react-native%22+expo

We dont have a dedicated label for Expo but it falls under React Native.

Does that help?

**Reply by mwarger** [reply]
Thank you, that does help.  I still think it might be worth looking into changing the NX report output - at the least, there's unnecessary noise created, IMO.

**Comment by AgentEnder** [comment]
I think this should be fixed now - the report output no longer includes things which would just say "not found". We will still ask for it, and it may create noise (imagine things like @nx/jest showing up for angular specific repos), but its one of the more helpful things for people to provide when troubleshooting an issue.
